src/docs/sphinx/dev_guide/gpu_porting.rst

```diff
@@ -0,0 +1,302 @@
+.. ## Copyright (c) 2017-2022, Lawrence Livermore National Security,con LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _gpu-porting-label:
+
+*******************************
+GPU Porting in Axom
+*******************************
+
+Axom uses RAJA and Umpire as the main workhorses for GPU porting,
```

Comment:
Should probably add a link to RAJA and Umpire here.

Maybe something like:

Axom uses the following two libraries as the main workhorses for GPU porting:

* RAJA (<- make this a link): handles software abstractions that enable architecture and programming model portability for HPC applications 
* Umpire (<-again link): resource management library that allows the discovery, provision, and management of memory on machines with multiple memory devices like NUMA and GPUs.